### PR TITLE
"New" names

### PR DESCRIPTION
**Pull request by Stemby**
These cities changed the official name in 1995, 2001 and 1996.

### CODE REVIEW

**Comment by jjeffryes** [comment]
Good catch, @Stemby. Thank you.

**Comment by Stemby** [comment]
You are welcome, @jjeffryes.

I'd like to find the modified string in Transifex: what can I do?

Thanks!

**Comment by Stemby** [comment]
Up...

**Comment by jjeffryes** [comment]
The city names under timezones. The strings are numbers that correspond to the timezone, like timezones.-12, timezones.-11, etc.

File: js/models/timezonesMd.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ module.exports = Backbone.Model.extend({
             "offset": 5
         },
         {
-            "name":"(GMT +5:30) Bombay, Calcutta, Madras, New Delhi",
+            "name":"(GMT +5:30) Mumbai, Kolkata, Chennai, New Delhi",
```
Comment (rmisio) [inline js/models/timezonesMd.js:91]: +1 for not using the imperial names!

**Comment by Stemby** [comment]
Up: on Transifex I still find the old names.

Thanks!

**Pull request merged**